src/test/java/org/springframework/data/hazelcast/repository/query/HazelcastSortAccessorTest.java

```diff
@@ -0,0 +1,146 @@
+package org.springframework.data.hazelcast.repository.query;
+
+import com.hazelcast.query.Predicate;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.history.RevisionSort;
+import org.springframework.data.keyvalue.core.query.KeyValueQuery;
+import org.springframework.util.ObjectUtils;
+
+import java.io.Serializable;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class HazelcastSortAccessorTest {
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test()
+    public void test1() {
```

Comment:
Please give meaningful names to tests so that they describe scenarios that test cases represent

Reply:
yes, done, gave appropriate names